Tests/src/PublicAPITests/ECommerceSystemTests.cpp

```diff
@@ -136,4 +136,31 @@ CSP_PUBLIC_TEST(CSPEngine, ECommerceSystemTests, GetProductInformationTest)
 
 	LogOut(UserSystem);
 }
-#endif
\ No newline at end of file
+#endif
+
+#if RUN_ECOMMERCE_TESTS || RUN_ECOMMERCE_GET_CHECKOUT_INFORMATION_TEST
+CSP_PUBLIC_TEST(CSPEngine, ECommerceSystemTests, GetCheckoutInformationTest)
```

Comment:
Can we also test unhappy paths here? Like what happens if an invalid space ID is passed in, other invalid parameters etc

Reply:
sorted